Automattic/jetpack
# Sync: Add Atomic Constant

We could use the Atomic client constant on WP.com in determining when some JITMs should fire. This adds that constant (and sorts all of the arrays while I'm touching it).

#### Changes proposed in this Pull Request:

* Add the `ATOMIC_CLIENT_ID` constant to the sync array.

#### Jetpack product discussion


p9dueE-3L6-p2 

#### Does this pull request change what data or activity we track or use?


We're adding a constant that is used on the Automattic-hosted platform, so it's already data known to a8c.


#### Testing instructions:
* Run this on an Atomic client.
* Check on WP.com if the constant is available.
